courses/catalog_serializers.py

```diff
@@ -71,12 +76,66 @@ def get_thumbnail_url(self, program):
         except ProgramPage.DoesNotExist:
             return None
 
+    def get_instructors(self, program):
+        """Get the list of instructors from the program page"""
+        try:
+            page = program.programpage
+        except ProgramPage.DoesNotExist:
+            return []
+
+        return list(page.faculty_members.values("name"))
+
+    def get_total_price(self, program):
+        """Get the combined price of all courses"""
+        return str(program.price * program.num_required_courses)
+
+    def get_start_date(self, program):
+        """Get the starting date of the first course in the program"""
+        course = program.course_set.order_by("position_in_program").first()
+        if not course:
+            return None
+        first_unexpired = first_matching_item(
+            course.courserun_set.all().order_by("start_date"), lambda run: run.is_unexpired
+        )
+        if not first_unexpired:
+            return None
+        return first_unexpired.start_date.isoformat() if first_unexpired.start_date else None
```

Comment:
this can just be:
```python
return first_unexpired.start_date
```
which will return a date string in a format like `2019-09-12T19:23:03Z`, matching the date format returned by the xPro API, instead of `2019-09-12T19:23:03+00:00`

Ditto for the other date fields

Reply:
Sure, will do